superset/assets/package.json

```diff
@@ -153,10 +156,10 @@
     "sinon": "^4.0.0",
     "style-loader": "^0.18.2",
     "transform-loader": "^0.2.3",
-    "uglifyjs-webpack-plugin": "^1.1.0",
-    "url-loader": "^0.6.2",
-    "webpack": "^3.8.1",
-    "webpack-manifest-plugin": "1.3.2",
-    "webworkify-webpack": "2.1.0"
+    "url-loader": "^1.0.1",
+    "webpack": "^4.6.0",
```

Comment:
Oh noice! I tried to do this in the past and failed :)

Reply:
there were several loaders that had to be swapped out to support it 👍